### DISCUSSION TITLE
conflict with controlnet in A1111 1.4

### DISCUSSION THREAD

**Discussion by howDoILearnPython** [volotat/SD-CN-Animation#172] [General]
Anyone experienced extension conflict error with controlnet after update this to version 1.4 of A1111?

i got some extension conflict error from controlnet extension during image generation.  After i disable CN-SD extension, the error message gone.  i'm not sure if this is caused by other extension of this extension??

**Comment by GevatterWuff** [comment]
I got the same issues. No ControlNet means this extension currently does not work for me.

**Comment by rickylars** [comment]
I had the same problem after updating.
I disabled SD-CN-Animation together with Config-Presets, then re-enabled SD-CN-Animation and it worked again

**Reply by GevatterWuff** [reply]
Where there any other steps you remember to get ControlNet working again with SD-CN enabled? Disabling/enabling/reinstalling did not work for me and I don't have the Config-Presets extension installed.

**Reply by ZonSoft** [reply]
Would be very grateful for precise steps to make this extension work without conflicts.

**Comment by rickylars** [comment]
hmmm, to be more precise I first disabled all the addons, then when I saw Controlnet worked again I re-enabled everything back besides  SD-CN-Animation and Config-Presets. Because Controlnet was still working I re-enabled SD-CN-Animation. I then closed the browser and the terminal, and it was working again.

**Reply by ZonSoft** [reply]
Config-Presets extension does not have any new requirements, SD-CN-Animation does. I believe you can enable Config-Presets now. But it's pure luck you've got SD-CN-Animation up and running - not reproducible.
